### PR TITLE
Add wrapper for package_python_2_7_wget_3_2

### PR DESCRIPTION
**Pull request by bebatut**

### CODE REVIEW

File: packages/package_python_2_7_wget_3_2/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+categories: 
+  - Tool Dependency Packages
+description: "Contains a tool dependency definition that downloads and compiles version 3.2 of python wget package"
+name: package_python_2_7_wget_3_2
+owner: iuc
```
Comment (bgruening) [inline packages/package_python_2_7_wget_3_2/.shed.yml]: Can you please add a `type: tool_dependency_definition`

**Comment by bgruening** [comment]
@yhoogstrate @gregvonkuster can anyone please upload these packages.
For some reason I get: `Exception: Attempting to create a repository with configured owner [iuc] that does not matche API user [1-204-10-1002].` over my remote connection.

File: packages/package_python_2_7_wget_3_2/.shed.yml (hunk 1 above)
Comment (bebatut) [inline packages/package_python_2_7_wget_3_2/.shed.yml]: Done

Comment (bgruening) [inline packages/package_python_2_7_wget_3_2/.shed.yml]: Please also for the others :)

Comment (bebatut) [inline packages/package_python_2_7_wget_3_2/.shed.yml]: Done and also for the merged ones

**Comment by gregvonkuster** [comment]
@bgruening @yhoogstrate I've uploaded package_python_2_7_fisher_0_1_4 and package_python_2_7_six_1_10_0 to the TTS and the MTS, but the remaining PRs have not yet been merged.  If there are no issues, let me know and I'll merge them and upload.

**Comment by yhoogstrate** [comment]
@bgruening This is the same issue I contacted you about. I think it is related to https://github.com/galaxyproject/planemo/issues/437
If you merge or agree on merging I will create a zip package and upload it via the web interface.

@gregvonkuster From which PR are those then I'll review them. It would have my preference to only upload to MTS after merge and preferably by the person who merges. Otherwise we might end up with tools in testing phase in the MTS and people complaining they're not working.

**Comment by gregvonkuster** [comment]
@yhoogstrate I uploaded the 2 packages from these respective PRs:
https://github.com/galaxyproject/tools-iuc/pull/732
https://github.com/galaxyproject/tools-iuc/pull/731
I believe @bgruening reviewed these and merged them, but could not upload so he asked you or I to do it.  Sorry if I screwed up.  ;)

**Comment by yhoogstrate** [comment]
@gregvonkuster awesome, you're a few steps ahead of me! I have uploaded the python wget package into the TTS, downloaded it via galaxy and tested it and it works. LGTM and feel free to merge and deploy at the MTS :+1:

**Comment by gregvonkuster** [comment]
Done -  https://toolshed.g2.bx.psu.edu/view/iuc/package_python_2_7_wget_3_2/c7aaab8b0202

**Comment by bgruening** [comment]
Thanks a lot!

**Pull request merged**